podium/urls.py

```diff
@@ -15,9 +15,13 @@
 """
 from django.conf.urls import url, include
 from django.contrib import admin
+from . import views
 
 
 urlpatterns = [
+
     url(r'^admin/', admin.site.urls),
     url(r'^talks/', include('podium.talks.urls')),
+    url(r'^', views.index, name='index'),
```

Comment:
The URL pattern for the index should be `r'^$'`  Without the `$`, this becomes a catch-all pattern for all possible unmatched URLs, i.e. /blahblahblah becomes an alias for the index page, when in reality it should cause a 404.